dist/tools/pyterm/pyterm

```diff
@@ -729,6 +738,12 @@ if __name__ == "__main__":
                         action="store_true",
                         help="toggles the DTR and RTS pin of the serial line "
                         "when connecting, default is not toggling the pins")
+    parser.add_argument("-ir", "--invert_RTS",
```

Comment:
I would prefer keeping a common style for command line arguments: `--invert-rts` / `--invert-dtr`

Reply:
Changed it!